### DISCUSSION TITLE
ios audio issue

### DISCUSSION THREAD

**Discussion by Coldplayer1995** [awslabs/amazon-kinesis-video-streams-producer-sdk-cpp#1114] [Q&A]
**Describe the bug**
Lots of audio issues over playing the stream in Video.js not a single problem in hls.js this only happens on safari browser
**To Reproduce**
1. Go to player view
2. Make sure you using safari
3. Change player from hls.js to Video.js
4. Audio is fragmented or not playing continuously 

**Expected behavior**
A stream should work as expected

**Platform (please complete the following information):**
 - OS: Mac ios

**Comment by disa6302** [comment]
@Coldplayer1995 , 

Can you provide details on your pipeline, the version of the SDK being used and the codecs you are using?
Do you also have a sample we can use to replicate the behavior?
